open_discussions/views.py

```diff
@@ -102,8 +108,50 @@ def index(request, **kwargs):  # pylint: disable=unused-argument
         "podcast_frontpage_enabled": features.is_enabled(features.PODCAST_FRONTPAGE),
     }
 
-    return render(
-        request, "index.html", context={"js_settings_json": json.dumps(js_settings)}
+    return render(request, "react.html", context=dict(js_settings=js_settings))
+
+
+def index(request, **kwargs):  # pylint: disable=unused-argument
+    """Render the react app"""
+    return _render_app(request)
+
+
+def channel_post(request, **kwargs):
+    """Render a channel post's page as long as it isn't removed"""
+    post_id = kwargs.get("post_id", None)
+    if not post_id:
+        raise Http404("No post specified")
+
+    post = get_object_or_404(Post, post_id=post_id)
```

Comment:
I noticed that the author of a removed post now gets a 404, but a removed comment is still reachable to the author via direct link.  Not sure if that really matters or not, all other non-admin users get a 404.

Reply:
To clarify, a removed comment on a non-removed post

Reply:
Hm, are you ok with me opening a separate issue for that since it'll involve a bunch more test reworking?

Reply:
Fine with me!